.circleci/config.yml

```diff
@@ -0,0 +1,24 @@
+version: 2
+
+jobs:
+
+  build:
+    docker:
+      - image: python:3.7-alpine
+    steps:
+      - checkout
+      - run: apk add git
+      - run: pip install -r requirements.txt
+      - run: echo 'Run the tests....'
+      - run:
+          name: Initialize Elastic Beanstalk
+          command:
+              eb init -p docker project-confero
```

Comment:
ooh, what does this do?

Reply:
configures the repository for deployment to Elastic Beanstalk. According to some documentation (though circleci doesn't have an official example of deploying to elastic beanstalk) it is not necessary to include this everytime when you are deploying an app that is not in a docker container. However, I think because we're using docker we need it to set up the file within the docker container each time we spin it up.